tests/integration/app-charm/src/charm.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# Copyright 2024 Ubuntu
```

Comment:
Why do we have `Ubuntu` instead of `Canonical`?

Reply:
Good catch, thanks. Just updated.